server/app.js

```diff
@@ -1,70 +1,29 @@
 const express = require('express');
 const bodyParser = require('body-parser');
-const path = require('path');
-const db = require('../db');
+const db = require('../db/index.js');
```

Comment:
you can just leave this as `require('../db')`

Reply:
nice. thanks!